pkg/irsa/capa/capa.go

```diff
@@ -61,7 +61,7 @@ func (s *Service) Reconcile(ctx context.Context, outRequeueAfter *time.Duration)
 
 	s.Scope.Logger().Info("Reconciling AWSCluster CR for IRSA")
 
-	b := backoff.NewMaxRetries(3, 5*time.Second)
+	b := backoff.NewMaxRetries(15, 5*time.Second)
```

Comment:
Can we write a small comment explaining why we need 15 seconds? That would prevent someone in the future trying to "optimize" it by mistake.